transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/CustomAttributesStoreImpl.kt

```diff
@@ -30,10 +40,27 @@ internal class CustomAttributesStoreImpl(
     }
 
     private fun isCustomAttributesValid(customAttributes: Map<String, String>): Boolean {
+        // Check if the size limit is unset, which means any size is valid
```

Comment:
I would say that comments here are redundant. Usually, we want to leave comments  when we have a weird logic or some workaround.